src/utils/Processing.ts

```diff
@@ -27,19 +27,30 @@ export function findInputDir(input: string): string {
     logger.info('path-to-fsh-defs defaulted to current working directory');
   }
 
+  // TODO: Legacy support. Remove when no longer supported.
   // Use fsh/ subdirectory if not already specified and present
   const fshSubdirectoryPath = path.join(input, 'fsh');
   if (fs.existsSync(fshSubdirectoryPath)) {
     input = path.join(input, 'fsh');
-    logger.info('fsh/ subdirectory detected and add to input path');
+    return input;
```

Comment:
I think this could be an appropriate place for a warning about how the `fsh` directory is deprecated, and they should switch to `input/fsh`.

Reply:
Ah good idea! I didn't even think about that.